index.html

```diff
@@ -9,51 +9,235 @@
+                    <li><img src="./images/Counter.png" alt="counter"/></li>
+                    <li class="year">2022</li>
+                </ul>
+                <p class="tonic-details">A daily selection of privately personalized reads: no accounts or sign-ups required</p>
+            </div>
+    
+            <ul class="programing-languages">
+                <li class="lang-list"><a href="#">html</a></li>
+                <li class="lang-list"><a href="#">css</a></li>
+                <li class="lang-list"><a href="#">javaScript</a></li>
+            </ul>
+    
+            <button type="button">See Project</button>
+        </div>
+
+        <!--Multi-post Availability Section-->
+        <div class="multi-post-availability padding">
+            <div class="availability-icon">
+                <img src="./images/Availability.png" alt="Portfolio">
+            </div>
+            <div class="tonic">
+                <h2>Multi-Post <br>Stories</h2>
+                <ul class="tonic-section-header">
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="./images/Counter.png" alt="counter" class="counter"/></li>
+                    <li class="back-end-dev">BackEndDev</li>
+                    <li><img src="./images/Counter.png" alt="counter"/></li>
+                    <li class="year">2022</li>
+                </ul>
+                <p class="tonic-details">A daily selection of privately personalized reads: no accounts or sign-ups required</p>
+            </div>
+    
+            <ul class="programing-languages">
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javaScript</a></li>
+            </ul>
+            <button type="button">See Project</button>
+        </div>
+
+        <!--Gymfit Section-->
+        <div class="Gymfit padding">
+            <div class="frontpage-icon">
+                <img src="./images/Gymfit.png" alt="Portfolio" class="frontpage-icon">
+            </div>
+            <div class="tonic">
+                <h2>Tonic</h2>
+                <ul class="tonic-section-header">
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="./images/Counter.png" alt="counter" class="counter"/></li>
+                    <li class="back-end-dev">Back End Dev</li>
+                    <li><img src="./images/Counter.png" alt="counter"/></li>
+                    <li class="year">2022</li>
+                </ul>
+                
+                <p class="tonic-details">A daily selection of privately personalized reads: no accounts or sign-ups required</p>
+            </div>
+    
+            <ul class="programing-languages">
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javaScript</a></li>
+            </ul>
+    
+            <button type="button">See Project</button>
```

Comment:
- [ ] Your buttons should also follow the button styling: Active, Hover and Pressed. Please check the design of the buttons below:
![image](https://user-images.githubusercontent.com/80612925/170781816-c77de1fb-4017-4f35-9005-907ce5c81a89.png)